src/update-feeds/update-feeds.ts

```diff
@@ -152,52 +151,34 @@ export const decodeBeaconValue = (encodedBeaconValue: string) => {
   return decodedBeaconValue;
 };
 
-export const getFeedsToUpdate = (
+// https://github.com/api3dao/airnode-protocol-v1/blob/fa95f043ce4b50e843e407b96f7ae3edcf899c32/contracts/api3-server-v1/DataFeedServer.sol#L132
+export const encodeBeaconValue = (numericValue: string) => {
+  const numericValueAsBigNumber = ethers.BigNumber.from(numericValue);
+
+  return ethers.utils.defaultAbiCoder.encode(['int256'], [numericValueAsBigNumber]);
+};
+
+export const getFeedsToUpdate = async (
   batch: ReadDapiWithIndexResponse[],
-  deviationThresholdCoefficient: DeviationThresholdCoefficient
-): UpdateableDapi[] =>
-  batch
-    .map((dapiInfo): UpdateableDapi | null => {
-      const beaconsSignedData = dapiInfo.decodedDataFeed.beacons.map((beacon) => getStoreDataPoint(beacon.beaconId));
-
-      // Only update data feed when we have signed data for all constituent beacons.
-      if (beaconsSignedData.some((signedData) => !signedData)) return null;
-
-      const beaconsDecodedValues = beaconsSignedData.map((signedData) => decodeBeaconValue(signedData!.encodedValue));
-      // Only update data feed when all beacon values are valid.
-      if (beaconsDecodedValues.includes(null)) return null;
-
-      // https://github.com/api3dao/airnode-protocol-v1/blob/fa95f043ce4b50e843e407b96f7ae3edcf899c32/contracts/api3-server-v1/DataFeedServer.sol#L163
-      const newBeaconSetValue = calculateMedian(beaconsDecodedValues.map((decodedValue) => decodedValue!));
-      const newBeaconSetTimestamp = calculateMedian(
-        beaconsSignedData.map((signedData) => ethers.BigNumber.from(signedData!.timestamp))
-      )!.toNumber();
-      const adjustedDeviationThresholdCoefficient = multiplyBigNumber(
-        dapiInfo.updateParameters.deviationThresholdInPercentage,
-        deviationThresholdCoefficient
-      );
-      if (
-        !checkUpdateConditions(
-          dapiInfo.dataFeedValue.value,
-          dapiInfo.dataFeedValue.timestamp,
-          newBeaconSetValue,
-          newBeaconSetTimestamp,
-          dapiInfo.updateParameters.heartbeatInterval,
-          adjustedDeviationThresholdCoefficient
-        )
-      ) {
-        return null;
-      }
+  deviationThresholdCoefficient: DeviationThresholdCoefficient,
```

Comment:
Mhh, why not just use `number`? I undertand why use the named types for chain ID, provider name, etc... But I don't see the benefit here.